tests/river/fixtures/mimic_mapping.json

```diff
@@ -1829,6 +1829,18 @@
           "definition_id": "Identifier",
           "input_groups": []
         },
+        {
+          "path": "generalPractitioner",
+          "slice_name": "",
+          "definition_id": "Reference",
+          "input_groups": []
+        },
+        {
+          "path": "link[0].other",
+          "slice_name": "",
+          "definition_id": "Reference",
+          "input_groups": []
+        },
```

Comment:
what does this test ?

Reply:
It tests that we do store where attributes that are references are located in the fhir doc
See `assert analysis.reference_paths == [["generalPractitioner"], ["link", "other"]]` in `test_analyze_mapping`